src/mixxxapplication.cpp

```diff
@@ -19,25 +19,25 @@
 
 // When linking Qt statically on Windows we have to Q_IMPORT_PLUGIN all the
```

Comment:
```suggestion
// When linking Qt statically we have to Q_IMPORT_PLUGIN all the
```
IIUC you are changing this code because this is needed on macOS, right? So I think it is needed when Qt is linked statically regardless of the OS?